connectivity/suite.go

```diff
@@ -130,6 +133,19 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 		return check.ResultDNSOKRequestDrop, check.ResultNone
 	})
 
+	// This policy allows UDP to kube-dns and port 80 TCP to all 'world' endpoints.
+	ct.NewTest("to-entities-world").
+		WithPolicy(clientEgressToEntitiesWorldPolicyYAML).
+		WithScenarios(
+			tests.PodToWorld(""),
+		).
+		WithExpectations(func(a *check.Action) (egress, ingress check.Result) {
+			if a.Destination().Port() == 80 {
+				return check.ResultOK, check.ResultNone
+			}
+			return check.ResultDrop, check.ResultNone
```

Comment:
Maybe add a comment that the scenario includes traffic to port 443 that will be verified to fail due to the `ResultDrop` return here?

Reply:
Added comment.